hack/functests.sh

```diff
@@ -89,15 +90,15 @@ if [ "$KUBEVIRT_E2E_PARALLEL" == "true" ]; then
 
     return_value=0
     set +e
-    functest --nodes=${KUBEVIRT_E2E_PARALLEL_NODES} ${parallel_test_args} ${KUBEVIRT_FUNC_TEST_GINKGO_ARGS}
+    functest --nodes=${KUBEVIRT_E2E_PARALLEL_NODES} ${parallel_test_args} ${KUBEVIRT_FUNC_TEST_GINKGO_ARGS} "${KUBEVIRT_FUNC_TEST_LABEL_FILTER}"
```

Comment:
If I understand correctly, previously, the tests with `Serial` tag were skipped on a parallel run and then focused on a serial run.  Now here a label filter is added in addition to those `--skip/--focus`. I wonder, how they coexist together? Are those AND'ed or OR'ed?

Reply:
`--skip/--focus` and `--label-filter` are AND'ed together by ginkgo (ref. https://onsi.github.io/ginkgo/#combining-filters)
The parallel and serial tests are still handled by `--focus/--skip` here
https://github.com/kubevirt/kubevirt/blob/b6fb0492142ae374c37e9098cabbba34997a89f8/hack/functests.sh#L84
So the result should be the same.
In conclusion, if no other `KUBEVIRT_E2E_FOCUS` or `KUBEVIRT_E2E_SKIP` are set externally, `--focus` and `--skip` should only contain `--focus=\\[Serial\\]` or `--skip=\\[Serial\\]` to distinguish serial/parallel execution.
Thanks

Reply:
Right, I must have missed that somehow from the docs. Makes sense, thank you for clarification.